subsys/mgmt/mcumgr/Kconfig

```diff
@@ -23,16 +254,18 @@ config MCUMGR_SMP_BT_AUTHEN
 	  Enables encrypted and authenticated connection requirement to
 	  Bluetooth SMP transport.
 
+endif # MCUMGR_SMP_BT
+
 config MCUMGR_SMP_SHELL
 	bool "Shell mcumgr SMP transport"
-	select MCUMGR
 	select SHELL
 	select SHELL_BACKEND_SERIAL
 	select BASE64
 	help
 	  Enables handling of SMP commands received over shell.  This allows
 	  the shell to be use for both mcumgr commands and shell commands.
 
+if MCUMGR_SMP_SHELL
```

Comment:
This is reduntant with `depends on MCUMGR_SMP_SHELL` in `MCUMGR_SMP_SHELL_MTU`.

Reply:
Done, thanks!